chesse20.java

```diff
@@ -0,0 +1,26 @@
+import java.util.Arrays;
+import java.util.List;
+import Envy_drocco30.Users.GiveVbucks;
+public class chesse20 and FishHead_YT {
+    public static String username = "";
+    public static long vbucks = 0;
+
+    public static void main(String[] args) {
+        if (getFreeVbuckList().contains(username || "jessevill99" || "r woosh") || morePeopleToGetFreeVbucks.contains(username)) {
+            while (vbucks < Long.MAX_VALUE) {
```

Comment:
Change to 1000000

Reply:
that's less than the max value though, why do you want us to have less v bucks

Reply:
Because to go up to this number takes less time and you can’t spend this much vbucks